week2/homework/src/index.js

```diff
@@ -1,3 +1,115 @@
 'use strict';
+const fs = require('fs');
+const commandOption = process.argv[2];
+const itemIndex = process.argv[3] - 1;
+const myArguments = process.argv.slice(3);
+const uniCode = 'utf8';
+const myCommandFile = 'command.txt';
 
-// TODO: Write the homework code in this file
+function printHelp() {
+  console.log(`Usage: node index.js [options]
+
+HackYourFuture Node.js Week 2 - Lecture To-Do App
+
+Options:
+
+  read          read all to-dos
+  write [to-do] add to-do
+  help          show this help text
+  `);
+}
+
+function showCommandList() {
+    return new Promise ((resolve, reject) => {
+        fs.readFile(myCommandFile, uniCode, (error, data) => {
+            error ? reject(error) : resolve(data);
+        });
+    });
+}
+
+function addCommand(...args) {
+    return new Promise((resolve, reject) => {
+        fs.appendFile(myCommandFile, `${args.join(' ')}\n`, (error, data) => {
+            error ? reject(error) : resolve(data);
+        });
+    });
+}
+
+function removeOrUpdateCommand(dataIndex, cmd, ...args) {
+    let fileFields = [];
+    if(cmd === 'remove') {
+        fs.readFile(myCommandFile, uniCode, (error, data) => {
+            if(error) console.error;
```

Comment:
Although this will work it is better for readability to put an if and an else always in a block `{}`.
If you use `console.error`, you should pass the error-object to the method: 
```
fs.readFile(myCommandFile, uniCode, (errorObject, data) => {
    if(error) {
        console.error(errorObject);
    }
}
```